Tasks/ContainerBuildV0/src/buildcontainer.ts

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+import tl = require('azure-pipelines-task-lib/task');
+import path = require('path');
+import docker = require("./docker");
+import buildctl = require("./buildctl");
+
+tl.setResourcePath(path.join(__dirname, '..', 'task.json'));
+   
+async function buildContainer() {
+    if(process.env["RUNNING_ON"] == "KUBERNETES") {
```

Comment:
who sets this variable?

Reply:
This variable is set inside the agentpod spawned in kubernetes cluster. This is just to differentiate if we are building using buildctl or the user is using normal hosted agents